presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -1545,6 +1554,17 @@ public HiveInsertTableHandle beginInsert(ConnectorSession session, ConnectorTabl
         return result;
     }
 
+    private HiveCompressionCodec getHiveCompressionCodec(ConnectorSession session, boolean isTemporaryTable, HiveStorageFormat storageFormat)
+    {
+        if (isTemporaryTable) {
+            return getTemporaryTableCompressionCodec(session);
+        }
+        if (storageFormat.equals(ORC) || storageFormat.equals(DWRF)) {
```

Comment:
use "=="

Reply:
@jainxrohit Would you address this comment?

Reply:
@bhhari I don't have a strong preference on equals vs == for enums, even though both do the same job. I find equals method less distracting as its the one gets used for other types. Let me know if you have strong preference.

Reply:
@jainxrohit I think this should use `==` as other places in that file use that.

Reply:
Ah, make sense, I will change it.